src/components/calcite-value-list-item/calcite-value-list-item.tsx

```diff
@@ -81,22 +96,51 @@ export class CalciteValueListItem {
 
   getPickListRef = (el) => (this.pickListItem = el as HTMLCalcitePickListItemElement);
 
+  handleKeyDown = (event: KeyboardEvent) => {
+    if (event.key === " ") {
+      this.handleActivated = !this.handleActivated;
+    }
+  };
+
+  handleBlur = () => {
+    this.handleActivated = false;
+  };
+
   // --------------------------------------------------------------------------
   //
   //  Render Methods
   //
   // --------------------------------------------------------------------------
 
+  renderHandle() {
+    const { icon } = this;
+    if (icon === ICON_TYPES.grip) {
+      return (
+        <a
```

Comment:
fixme: Can you make this a `div` with a button role?  I don't think an anchor makes sense semantically.

Reply:
Would an actual button be more appropriate?

Reply:
You could, but you'll have to stop the click event from triggering.

Reply:
outside of a form and with no click handler - would there be any effect?

Reply:
Clicking shouldn't trigger handle activation

Reply:
It won't because I'm listening to onKeyDown on the button to activate it.